src/apps/loan-list/modal/modals.test.tsx

```diff
@@ -63,80 +63,6 @@ describe("Modals", () => {
 
     cy.get(".modal-details__container").should("exist");
   });
-
-  it("It opens due date modal with query params", () => {
```

Comment:
Why do you delete the test?

Reply:
Because I removed some code that made the test work.